src/Integration/VsShellUtils.cs

```diff
@@ -20,10 +21,39 @@ namespace SonarLint.VisualStudio.Integration
 {
     public static class VsShellUtils
     {
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability",
+            "S2223:Non-constant static fields should not be visible",
+            Justification = "Internal for testing purposes. Cannot make const. Cannot make readonly as is being passed by ref.",
+            Scope = "member",
+            Target = "~F:SonarLint.VisualStudio.Integration.VsShellUtils.SonarLintOutputPaneGuid")]
+        internal static Guid SonarLintOutputPaneGuid = new Guid("EB476B82-D73A-44A6-AFEF-830F7BBA73DB");
+
+        /// <summary>
+        /// Writes a message to the SonarLint output pane. Will append a new line after the message.
+        /// </summary>
+        public static void WriteToSonarLintOutputPane(IServiceProvider serviceProvider, string messageFormat, params object[] args)
+        {
+            if (serviceProvider == null)
+            {
+                throw new ArgumentNullException(nameof(serviceProvider));
+            }
+
+            if (messageFormat == null)
+            {
+                throw new ArgumentNullException(nameof(messageFormat));
+            }
+
+            IVsOutputWindowPane sonarLintPane = GetOrCreateSonarLintOutputPane(serviceProvider);
+            if (sonarLintPane != null)
+            {
+                WriteLineToPane(sonarLintPane, messageFormat, args);
+            }
+        }
+
         /// <summary>
         /// Writes a message to the general output pane. Will append a new line after the message.
         /// </summary>
-        public static void WriteToGeneralOutputPane(IServiceProvider serviceProvider, String messageFormat, params object[] args)
+        public static void WriteToGeneralOutputPane(IServiceProvider serviceProvider, string messageFormat, params object[] args)
```

Comment:
Removed. Also removed the no longer used `VsGeneralOutputWindowNotifer` and associated tests.